sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
+    --------
+
+    KMeans:
```

Comment:
There should be a space before the column for the rendering to be correct. I also don't think it allows for multiline description. See the following link for numpydoc's documentation: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt